gammapy/data/observations.py

```diff
@@ -0,0 +1,569 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import numpy as np
+from collections import OrderedDict
+from astropy.coordinates import SkyCoord
+from astropy.units import Quantity
+from astropy.utils import lazyproperty
+from ..extern.six.moves import UserList
+from ..irf import EnergyDependentTablePSF, PSF3D, IRFStacker
+from ..utils.energy import Energy
+from ..utils.fits import earth_location_from_dict
+from ..utils.table import table_row_to_dict
+from ..utils.time import time_ref_from_dict
+
+__all__ = [
+    'ObservationIACT',
+    'DataStoreObservationIACT',
+    'ObservationList',
+]
+
+log = logging.getLogger(__name__)
+
+
+class ObservationIACT(object):
+    """Container class for an IACT observation
+
+    Parameters follow loosely the "Required columns" here:
+    http://gamma-astro-data-formats.readthedocs.io/en/latest/data_storage/obs_index/index.html#required-columns
+
+    Parameters
+    ----------
+    obs_id : int
+        Observation ID
+    gti : `~gammapy.data.GTI`
+        Good Time Intervals, see: http://gamma-astro-data-formats.readthedocs.io/en/latest/events/gti.html
```

Comment:
Suggest to remove all the URLs to the spec, and just link to it once in the class-level docstring.

Reply:
Done.
Although i think there are never too many links to the documentation :) But this docstring definitely looked a bit messy.